Windows API change on GetCurrentProcess is breaking the lib

On windows 10, just get the last release: 
> go get github.com/prometheus/client_golang/prometheus/promauto
> C:\...\github.com\prometheus\client_golang\prometheus\process_collector_windows.go:78:9: assignment mismatch: 2 variables but "golang.org/x/sys/windows".GetCurrentProcess returns 1 values

It seems related to this change: https://github.com/golang/sys/commit/2dccfee4fd3e23e10ba17b9d696cf2388c28764b
There are other lib broken because of this change: https://github.com/golang/go/issues/34461

go version go1.12.4 windows/amd64

--- a/go.mod
+++ b/go.mod
@@ -4,10 +4,10 @@ require (
 	github.com/beorn7/perks v1.0.1
 	github.com/golang/protobuf v1.3.2
 	github.com/json-iterator/go v1.1.7
-	github.com/prometheus/client_model v0.0.0-20190129233127-fd36f4220a90
+	github.com/prometheus/client_model v0.0.0-20190812154241-14fe0d1b01d4
 	github.com/prometheus/common v0.7.0
-	github.com/prometheus/procfs v0.0.3
-	golang.org/x/sys v0.0.0-20190801041406-cbf593c0f2f3
+	github.com/prometheus/procfs v0.0.5
+	golang.org/x/sys v0.0.0-20190927073244-c990c680b611
 )
 
 go 1.11

